chain-tx-enclave/tx-validation/app/src/enclave_u/mod.rs

```diff
@@ -165,9 +166,9 @@ pub fn check_tx(
                         },
                     ) => Some(StakedState::new_init(
                         deposit_amount,
-                        request.info.previous_block_time,
+                        None,
                         to_staked_account,
-                        true,
+                        &StakedStateDestination::UnbondedFromCustomTime(request.info.previous_block_time),
```

Comment:
bug discovered in integration test -- this should be (I guess):
```suggestion
                        &StakedStateDestination::Bonded,
```